core/src/main/java/de/bwaldvogel/mongo/oplog/OperationType.java

```diff
@@ -0,0 +1,11 @@
+package de.bwaldvogel.mongo.oplog;
+
+public enum OperationType {
+    DELETE("d"),
+    UPDATE("u"),
+    INSERT("i");
+
+    private final String value;
```

Comment:
instead of `value` I would call it `code` / `wireValue` (or something else)
Enum has implicit "value" already

Reply:
:+1: to call it `code` instead of `value` in this case.